db/migrate/20190724135655_add_enum_column_to_application_scope_limitations.rb

```diff
@@ -0,0 +1,5 @@
+class AddEnumColumnToApplicationScopeLimitations < ActiveRecord::Migration[5.2]
+  def change
+    add_column :application_scope_limitations, :substantive, :boolean, default: true
```

Comment:
why creating a new migration rather than sticking this in the previous migration when this table gets created?